newroutes/__tests__/NetlifyToml.spec.js

```diff
@@ -0,0 +1,45 @@
+const express = require("express")
+const request = require("supertest")
+
+const { errorHandler } = require("@middleware/errorHandler")
+const { attachReadRouteHandlerWrapper } = require("@middleware/routeHandler")
+
+const { NetlifyTomlRouter } = require("../netlifyToml")
+
+describe("NetlifyToml Router", () => {
+  const mockNetlifyTomlService = {
+    read: jest.fn(),
+    update: jest.fn(),
+  }
+
+  const router = new NetlifyTomlRouter({
+    netlifyTomlService: mockNetlifyTomlService,
+  })
+
+  const app = express()
+  app.use(express.json({ limit: "7mb" }))
+  app.use(express.urlencoded({ extended: false }))
+
+  // We can use read route handler here because we don't need to lock the repo
+  app.get("/netlifyToml", attachReadRouteHandlerWrapper(router.readNetlifyToml))
+  app.use(errorHandler)
+
+  const accessToken = undefined // Can't set request fields - will always be undefined
+
+  const reqDetails = { accessToken }
+
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  describe("readNetlifyToml", () => {
+    const netlifyTomlHeaderValues = "netlifyTomlHeaderValues"
+    mockNetlifyTomlService.read.mockResolvedValue(netlifyTomlHeaderValues)
+    it("retrieves netlifyToml details", async () => {
```

Comment:
could we have a line spacing here for readability?

Reply:
[9a6cbb5](https://github.com/isomerpages/isomercms-backend/pull/357/commits/9a6cbb53dc4fdc4de1e57ed72614e48e9f62ba71)